xmtp_id/src/verifier.rs

```diff
@@ -0,0 +1,47 @@
+use ethers::types::{Address, BlockId, Signature};
+use ethers::providers::{Http, Middleware, Provider};
+use std::convert::TryFrom;
+use std::sync::Arc;
+use crate::bindings::coinbase_smart_wallet::CoinbaseSmartWallet;
```

Comment:
Where are the bindings being generated? Is this a manual generation or is there a script somewhere?

in `didethresolver`, we used an `abigen!` macro tied to just the top-level `DIDRegistry` smart contract (despite DIDRegistry pulling in a bunch of dependencies), which was enough to let us use the DIDRegistry struct and call into the smart contract from rust code. 

Is that not possible here? I would rather avoid adding a bunch of bindings we won't use

Reply:
The bindings are generated by `forge find` in the coinbase smart wallet repo. I didn't use `abigen!` because it is not supported well by my editor but I have no problems using it.

Reply:
tysm!My beef was mainly because I can't checkout the abigen'ed code in my editor.

Don't want to clutter the codebase either, using abigen! now.

Reply:
ah ok, yeah you can use `cargo doc --document-private-items` to inspect the abigen!'ed code